hack/render-manifests.go

```diff
@@ -16,23 +16,29 @@ func exitWithError(err error, cause string, args ...interface{}) {
 
 func main() {
 	type Inventory struct {
```

Comment:
The render-manifests.go will be used only for operator now, since operator is going to be in charge of render operand.